doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -144,7 +144,11 @@ Gazell
 Matter
 ------
 
-* Added a new documentation page :ref:`ug_matter_group_communication` in the :ref:`ug_matter_intro_overview`.
+* Added:
```

Comment:
You have also updated west zap-gui command, so it's worth to mention that is has new arguments.